02-peer-protocol.md

```diff
@@ -571,20 +570,6 @@ Closing happens in two stages:
 1. one side indicates it wants to clear the channel (and thus will accept no new HTLCs)
 2. once all HTLCs are resolved, the final channel close negotiation begins.
 
-        +-------+                              +-------+
```

Comment:
IMO we should have awareness of this be signalled via feature bit rather than remove the old version. Then in the future, we can start to make the bit required to enforce the switch over on the protocol level.